xbmc/addons/Skin.cpp

```diff
@@ -27,6 +27,11 @@
 #include "filesystem/Directory.h"
 #include "filesystem/File.h"
 #include "filesystem/SpecialProtocol.h"
+
+#include "guilib/GUIControlFactory.h" // for colors
```

Comment:
Agreed. Not even sure if this include is needed at all, seems to build fine without it (at least on linux)
Might be some relict from the old PR.